docs/en/04_Changelogs/4.10.0.md

```diff
@@ -6,6 +6,18 @@
 
 ### Other new features
 
+#### New `SS_TEMP_PATH` environment variable
+
+This release adds support for a new `SS_TEMP_PATH` environment variable, which allows developers to control where a series of “temporary” files are created. These include [Manifests](/developer_guides/execution_pipeline/manifests), [Object Caching](/developer_guides/performance/caching) and [Partial Template Caching](/developer_guides/templates/partial_template_caching).
+
+The environment variable can be set to either an absolute path which must have a leading `/` (e.g. `SS_TEMP_PATH='/tmp/silverstripe/cachedir'`), or a path relative to the project root (e.g. `SS_TEMP_PATH='silverstripe-cache'`). See the [environment management](/getting_started/environment_management) section for more information.
+
+**Please note:** setting this environment variable will impact both requests served by a webserver (i.e. a typical website pageview) and command-line execution. If your webserver user and command-line user are different, or have different filesystem permissions, you may encounter problems when using this setting.
+
+The new `SS_TEMP_PATH` environment variable replaces the similarly-named `TEMP_PATH` environment variable, which only ever offered partial support for adjusting the temporary files' location. This is because setting a `TEMP_PATH` _environment variable_ would affect [Injector](/developer_guides/extending/injector) service definitions, but would **not** affect code that referenced the `TEMP_PATH` _PHP constant_. The confusion with both environment variables and PHP constants having the same name is why the environment variable has been renamed and prefixed with `SS_`, in-keeping with other environment variables.
```

Comment:
```suggestion
The new `SS_TEMP_PATH` environment variable replaces the similarly-named `TEMP_PATH` _environment variable_, which only ever offered partial support for adjusting the temporary files' location. This is because setting a `TEMP_PATH` _environment variable_ would affect [Injector](/developer_guides/extending/injector) service definitions, but would **not** affect code that referenced the `TEMP_PATH` _PHP constant_. The confusion with both environment variables and PHP constants having the same name is why the _environment variable_ has been renamed and prefixed with `SS_`, in-keeping with other environment variables.
```

Reply:
@kinglozzer Just to confirm, what happens if your site has an existing TEMP_PATH environment variable?  Will your cache directory change? Or will it remain the same? It's worth mentioning that in the changelog so people know what to expect

Reply:
Nothing will change for people using `TEMP_PATH` environment variable, I’ve added an extra sentence to the start of the last paragraph to clarify that